python/setup.py

```diff
@@ -420,7 +409,7 @@ def build_extensions(self):
     long_description=open(os.path.join(PYNGRAPH_ROOT_DIR, 'README.md')).read(),
     long_description_content_type='text/markdown',
     ext_modules=ext_modules,
-    package_dir=package_dir,
+    package_dir={'': 'src'},
```

Comment:
Does this `'' ` empty quote mean no name for package?  Earlier there was `'ngraph.impl.onnx_import'`